libafl/src/stages/tmin.rs

```diff
@@ -349,33 +349,20 @@ where
 /// provided
 #[derive(Clone, Debug)]
 pub struct MapEqualityFeedback<M, S> {
-    name: String,
-    obs_name: String,
+    name: Cow<'static, str>,
+    obs_name: Cow<'static, str>,
     orig_hash: u64,
     phantom: PhantomData<(M, S)>,
 }
 
-impl<M, S> MapEqualityFeedback<M, S> {
```

Comment:
this guy got deleted for mistake?

Reply:
Nope, not accidental -- this one should not be publicly exposed.